lib/openstack_query/query_blocks/query_executer.py

```diff
@@ -29,7 +31,11 @@ def __init__(self, prop_enum_cls: Type[PropEnum], runner_cls: Type[RunnerWrapper
         self.runner = runner_cls(self._prop_enum_cls.get_marker_prop_func())
         self._client_side_filters = None
         self._server_side_filters = None
-        self._raw_results = []
+        self._results = []
+
+        # for chaining
+        self._forwarded_values = {}
+        self._link_prop_func = lambda a: None
```

Comment:
Can't we just store this? Instead of passing a lambda in lets just do

```
        current_query.executer.append_forwarded_results(
            new_query.group_by(link_props[1]).to_props(),
            self._link_prop = parse_property(link_props[0], obj),
        )
```

Reply:
I think i've addressed this since we're not storing link_prop in executer anymore